src/com/untamedears/JukeAlert/model/LoggedAction.java

```diff
@@ -22,7 +22,8 @@
     BLOCK_USED(9),
     LOGIN(10),
     LOGOUT(11),
-    EXCHANGE(12);
+    EXCHANGE(12),
+    CART_DESTROY(13);
```

Comment:
Let's use VEHICLE_DESTROY, since this will catch any vehicle like horses and hopper carts.